app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseScreen.kt

```diff
@@ -77,11 +72,8 @@ abstract class BaseScreen<K : ScreenKey, B: ViewBinding, M : Parcelable, E, F, R
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
 
-    val uiRenderer = uiRenderer()
-    controller.connect(
-        Connectables.contramap({ it })
-        { eventConsumer -> connectViews(eventConsumer, uiRenderer) }
-    )
+    val rxBridge = RxMobiusBridge(events(), uiRenderer())
+    controller.connect(Connectables.contramap({ it }, rxBridge))
```

Comment:
Same comment as above, should we stop using `contramap`? we can re-introduce it if needed. At this point it seems like we are not modifying the model data before rendering anywhere.